docs/diagrams/StorageLoad.puml

```diff
@@ -0,0 +1,75 @@
+@startuml
+!include style.puml
+
+title Main Structure of Storage Load\n
+
+participant ":FitTrack" as main MODEL_COLOR
+participant ":Storage" as storage STORAGE_COLOR
+participant "ui: Ui" as ui UI_COLOR
+participant ":UserProfileDecoder" as pDecoder STORAGE_COLOR_T1
+participant ":MealListDecoder" as mDecoder STORAGE_COLOR_T2
+participant ":WorkoutListDecoder" as wDecoder STORAGE_COLOR_T3
+
+main -> main ++: start()
+activate main MODEL_COLOR
```

Comment:
I like the colored representation. It allows readers to differentiate each class more easily.